headhunter/data/ir_attachment_post.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
```

Comment:
This should be noupdate. Please have a look at the warnings.

Reply:
okay